src/components/molecules/NavMenu/index.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react'
+import { StyleMenuLinkContainer } from "./styles"
+import translationsEn from "../../../../config/translations/en.json"
+
+import MenuLink from "../../atoms/MenuLink/index"
+
+const Links = translationsEn.menu;
+
+const NavMenu: React.FC = () => {
+    return (
+        <StyleMenuLinkContainer>
+            {Links.map(link => (
+                <MenuLink linkName={link.title} />
```

Comment:
you forgot the KEY hahahaha

You probably need to add a memo in front of your computer or idk. You always forgot the key and you'll have a lot of bugs if you don't add the key.

Reply:
AAHHHH! 🤣 I'm gonna put a sticky note on my monitor right now! lol